Sources/MapboxMaps/Gestures/GestureHandlers/PinchBehaviors/PanRotatePinchBehavior.swift

```diff
@@ -1,3 +1,5 @@
+import CoreGraphics
```

Comment:
nit: I'm not familiar with swift, but is it needed as there's no other change in this class besides this import.

Reply:
Xcode has started auto-importing frameworks which is leading to a lot of things like this happening. Probably there were other changes to this file in a previous iteration of this PR which led to this import which was then not reverted.

Reply:
@macdrevx exactly! I'm willing to keep it, Xcode has a point - we do use `CoreGrapics` types here.